packages/slice-machine/lib/utils/str.ts

```diff
@@ -1,6 +1,14 @@
 import camelCase from "lodash/camelCase";
 
 const camelizeRE = /-(\w)/g;
+
+export function transformKeyAccessor(str: string): string {
```

Comment:
Maybe we should this function in other frameworks no ? I think the problem can occur in React and Vue also no ?